test/integration_test.py

```diff
@@ -212,28 +211,34 @@ def managed_access_sources_by_catalog(self) -> Dict[CatalogName,
                     managed_access_sources[catalog].add(ref)
         return managed_access_sources
 
+    def _partition_bundles(self,
+                           catalog: CatalogName,
+                           source: str
+                           ) -> List[SourcedBundleFQID]:
+        """
+        Select a partition of bundles from the specified source and verify that
```

Comment:
```suggestion
        Randomly select a partition of bundles from the specified source, checks that
```

Reply:
I believe it should be "check" not "checks".